Makefile

```diff
@@ -305,27 +305,31 @@ create-cluster: $(CLUSTERCTL) ## Create a development Kubernetes cluster on AWS
 	# Install cert manager.
 	kubectl \
 		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
-		create -f examples/_out/cert-manager.yaml
-	# Wait for cert-manager pods to be created
-	sleep 20
-	# Wait for cert-manager pods to be ready.
+		apply -f examples/_out/cert-manager.yaml
+	# Wait for webhook servers to be ready to take requests
 	kubectl \
 		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
-		wait --for=condition=Ready --namespace=cert-manager --timeout=15m pods --all
-	# Wait for webhook servers to be ready to take requests
-	sleep 10
+		wait --for=condition=Available --timeout=5m apiservice v1beta1.webhook.cert-manager.io
 	# Apply provider-components.
 	kubectl \
 		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
-		create -f examples/_out/provider-components.yaml
+		apply -f examples/_out/provider-components.yaml
+	# Wait for CAPI pod
+	kubectl \
+		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
+		wait --for=condition=Ready --timeout=5m -n capi-system pod -l control-plane=cluster-api-controller-manager
```

Comment:
Would it be better to wait for the api types or the webhooks instead?

Reply:
We could add waits on the CRDs too. I couldn't find a way to wait for a webhook to be ready - it's served by the pod, via the service. So if the pod is ready, there's maybe a slim chance that the webhook service isn't quite there yet, but if we start seeing flakes, we can add more waits here.

Reply:
@ncdc isn't waiting for the webhook service to be available precisely what the wait we are doing in the cert manager wait above?

Reply:
ah, nm, it's just checking the status of the CRD being registered...  I'm wondering if we could check on the status of the `Service` we deploy for the webhook instead.

Reply:
Not exactly. The cert-manager wait is waiting on an apiservice that comes from an aggregated api server, which means it's waiting to be able to talk to the service->pod for it.

Reply:
If we waited on the v1alpha3.cluster.x-k8s.io apiservice, you'd see this:
```
    "status": {
        "conditions": [
            {
                "lastTransitionTime": "2019-12-04T15:11:02Z",
                "message": "Local APIServices are always available",
                "reason": "Local",
                "status": "True",
                "type": "Available"
            }
        ]
    }
```

Reply:
The most extensive check would probably be to have liveness and readiness probes in the capi/capa pods, and then check that the 2 services have at least 1 endpoint.

Reply:
I think this is "good enough" for now. We can tweak later if needed.